auto-value-parcel/src/test/java/com/ryanharter/auto/value/parcel/model/SampleTypeWithParcelableContractSatisfied.java

```diff
@@ -0,0 +1,28 @@
+package com.ryanharter.auto.value.parcel.model;
```

Comment:
This is copying the format of your 'throwsForNonParcelableProperty' and 'acceptsParcelableProperties' tests. Happy to change if you have a suggestion as for how. I'm not sure how to check the lack of parcelable generation though. I don't want to test the output of AutoValue itself, because that's all implementation details that you don't control.